src/views/mypage/MyPageRecordView.vue

```diff
@@ -97,4 +87,30 @@ function prevPage() {
 function nextPage() {
   if (canGoNext.value) page.value--;
 }
+
+//한 페이지에 표시할 기록들 배열에 추구미 이미지 경로 추가
+const pagedRecordsWithImages = computed(() =>
+  pagedRecords.value.map(record => {
+    const choogoomi = CHOOGOOMI_MAP.find(
+      c => c.choogoomiName === record.choogooMi
+    );
+    const imageUrl = new URL(choogoomi.userLevel[0].character, import.meta.url)
+      .href;
+
+    return {
+      ...record,
+      imageUrl,
+    };
+  })
+);
+
+onMounted(async () => {
+  try {
+    const { data } = await axiosInstance.get('api/ranking/history');
```

Comment:
```suggestion
    const { data } = await axiosInstance.get('/api/ranking/history');
```
'api/~'보다는 '/api/~'로 수정하는게 좋을 것 같아요!